source/functions/BEMIO/writeH5/h5bemWriteParameters.m

```diff
@@ -1,20 +1,20 @@
-function h5bem_writeParameters(filename, wave_dir, water_depth, w, T)
+function h5bemWriteParameters(filename, waveDir, waterDepth, w, T)
 	% Function to write the simulation parameters to an existing 
     %     WEC-Sim/BEMIO-formatted h5 file
```

Comment:
I think this needs to be removed, and has been replaced by ``source/functions/BEMIO/writeH5.m``